cxx-checks/src/main/java/org/sonar/cxx/checks/CheckList.java

```diff
@@ -76,7 +76,8 @@ private CheckList() {
       UsingNamespaceInHeaderCheck.class,
       SafetyTagCheck.class,
       UseCorrectIncludeCheck.class,
-      XPathCheck.class,
+      // XPathCheck.class, this gives class not found exception in 5.6
+      // XPath no longer found in other plugins
```

Comment:
@guwirth supose it needs to be taken out completly, ive commented out just in case someone is actually using it. and if so perhaps try to port it? ive just looked at our instance and we havent used this. #908 raise to track this